test/Microsoft.AspNet.Mvc.TagHelpers.Test/CacheTagHelperTest.cs

```diff
@@ -320,6 +318,47 @@ public void GenerateKey_WithMultipleVaryByOptions_CreatesCombinedKey()
         }
 
         [Fact]
+        public async Task ProcessAsync_DoesNotUseCache_OnPassingFalseWhenEnabled()
+        {
+            // Arrange
+            var cache = new Mock<IMemoryCache>();
+            cache.Setup(c => c.Set(
+                /*key*/ It.IsAny<string>(),
+                /*link*/ It.IsAny<IEntryLink>(),
+                /*state*/ It.IsAny<object>(),
+                /*create*/ It.IsAny<Func<ICacheSetContext, object>>()))
+                .Returns(new DefaultTagHelperContent().SetContent("ok"))
+                .Verifiable();
+            object cacheResult;
+            cache.Setup(c => c.TryGetValue(It.IsAny<string>(), It.IsAny<IEntryLink>(), out cacheResult))
+                .Returns(false);
+            var tagHelperContext = GetTagHelperContext("unique-id", "original-child-content");
+            var tagHelperOutput = new TagHelperOutput("cache", new Dictionary<string, object>());
+            var cacheTagHelper = new CacheTagHelper
+            {
+                ViewContext = GetViewContext(),
+                MemoryCache = cache.Object,
+                Enabled = true
```

Comment:
OIC.  this is a bizarre test.  highly unlikely `Enabled` will change after initialization, never after `ProcessAsync` has run.